mono/metadata/mono-debug-debugger.h

```diff
@@ -20,7 +20,7 @@ mono_debug_get_seq_points (MonoDebugMethodInfo *minfo, char **source_file, GPtrA
 MONO_API void
 mono_debug_free_locals (MonoDebugLocalsInfo *info);
 
-MONO_API void
```

Comment:
This should probably be moved to mono-debug.h, and we should get rid of mono-debug-debugger.h, although this is a different issue than this PR.